setup.cfg

```diff
@@ -42,6 +42,7 @@ install_requires =
     httpx ~= 0.16
     importlib_metadata; python_version <'3.8'
     ordered-set
+    pip >= 21.1
```

Comment:
pip should not be a direct dependency. If this is about safety complaining, we need to rather set this inside tox' dependencies.

Reply:
Ah, yes - this didn't feel right when I did it.  Will look to do that instead.